data/hlint.yaml

```diff
@@ -334,14 +335,14 @@
 
     - warn: {lhs: id *** g, rhs: second g}
     - warn: {lhs: f *** id, rhs: first f}
-    - ignore: {lhs: zip (map f x) (map g x), rhs: map (f Control.Arrow.&&& g) x}
-    - ignore: {lhs: "\\(x,y) -> (f x, g y)", rhs: f Control.Arrow.*** g}
-    - ignore: {lhs: "\\x -> (f x, g x)", rhs: f Control.Arrow.&&& g}
-    - ignore: {lhs: "\\(x,y) -> (f x,y)", rhs: Control.Arrow.first f}
-    - ignore: {lhs: "\\(x,y) -> (x,f y)", rhs: Control.Arrow.second f}
-    - ignore: {lhs: "(f (fst x), g (snd x))", rhs: (f Control.Arrow.*** g) x}
     - hint: {lhs: "(fst x, snd x)", rhs:  x, note: DecreasesLaziness, name: Redundant pair}
 
+    # BIFUNCTOR
+
+    - hint: {lhs: "\\(x,y) -> (f x, g y)", rhs: bimap f g, note: IncreasesLaziness}
```

Comment:
Can you make sure it says `Data.Bifunctor.bimap` - that way HLint will be able to suggest the module name too. Ditto for the others below.

Reply:
Does that just need to be changed in the hints themselves, or do I need to change the import to qualified too?

Reply:
Just the hints themselves.